Feedback From Qingqing

comeseemeintheocean opened the pull request:
Hello @JimmyYang917 ,

This is my initial feedback about your research project. 

Your research proposal has provided the readers with a comprehensive selection of variables which relation to macro econometric analysis, you have considered the selection of variables with respect to many factors including the subject of the study itself and the external environment that can influence it, also gives me a hint to consider my variable selection.

However, there may also need some improvement towards the data cleaning part. The first is about the formatting under the ‘Data properties’, you have given a very detailed explanation and therefore take up a lot of space, giving some blank space under some subheadings I think would increase readability. The second is that we may can shorten the names of some variables, which is more common for example, treasurybills can be written as tbills. The third is I have personally taken the log transformation of some of my data to narrow the data range and get rid of the effect of outliers and a code loop may be more efficient. The fourth is for correlation plot, since we may only focus on the correlation between Y and X variables which means we can transfer the plot to the numeric value which only contains the correlation between Y and X variables and become more straightforward. The fifth is about the adf test, we may can write in a loop to show the results more efficient. 

I have made some changes in the qmd file already and hope it is making sense and helpful for you.

Good luck!
Qingqing Pang

comeseemeintheocean committed:
Improvements on data cleaning

JimmyYang917 commented:
Hi  Qingqing @comeseemeintheocean:

Thanks for the pull request and hope you have a great day.

After check all requirements and pull request details, I'd like to say thank you to what you have done and uopdate following changes.

- [x] update new adf test details and results

For repo designed in first stage recently, all variables transformation will be done after ADF tests results. Correlation table is quite clearly and nice, and just keep it for right now. Blank lines and shorter variable names will included in after wedsite design.

Thank you so much

Best wishes

Yuqiao

JimmyYang917 commented:
Website format update:

- [x] Rename the variables structure and discribtion
- [x] Update some Title structure

JimmyYang917 committed:
Updata ADF test

JimmyYang917 committed:
Delate the log transformation

Keep all data at basic form first, it keeps all informations and wait for further test.

donotdespair commented:
Nice cooperative work @comeseemeintheocean and @JimmyYang917  Thank you!
